app/javascript/components/ActionItemCard/index.js

```diff
@@ -70,6 +76,44 @@ function ActionItemCard({
     </Button>
   );
 
+  const formattedDueDate = () => {
+    if (formatDate) {
+      return formatDate(dueDate);
+    }
+    return dueDate;
+  };
+
+  // Handles logic for the first button on bottom of ActionItemCard
+  const renderFirstButton = () => {
```

Comment:
This method and the one below are a bit messy because of the button logic that ActionItemCards have to handle, with being rendered in multiple places. I added comments to try and make it more understandable.